crm_deduplicate_filter/readme/USAGE.rst

```diff
@@ -0,0 +1,11 @@
+To use this module, you need to:
+
+#. Go to *CRM/Sales > Tools > Deduplicate Contacts*.
+#. Mark "'Is a company?' field selected", "Parent company not set" or
+   "Parent company set (Contacts)" in the section 'Exclude contacts having'.
+#. This criteria will be used for excluding in the deduplication the selected
+   kind of records.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
Don't include this part, as this is old stuff that now it's auto-generated in the header.